devices/fields/spi/spi_v3_i2s.yaml

```diff
@@ -0,0 +1,45 @@
+I2SCFGR:
+  MCKOE:
+    Disabled: [0, Master clock output disabled]
+    Enabled: [1, Master clock output enabled]
+  ODD:
+    Even: [0, Real divider value is I2SDIV*2]
+    Odd: [1, Real divider value is I2SDIV*2 + 1]
+  I2SDIV: [0, 255]
```

Comment:
RM433 page 2233 mentions
> Bits 23:16 I2SDIV[7:0]: I2S linear prescaler
>I2SDIV can take any values except the value 1, when ODD is also equal to 1.

Is this something that changes anything?

Reply:
When range is defined svd2rust adds safe `fn set` in addition to `unsafe fn bits`.

But in this case it depends on other field state. So we should delete this.
I'm not sure.

Reply:
Ok, good no know. Thanks